lib/workers/branch.js

```diff
@@ -116,6 +117,22 @@ async function ensureBranch(upgrades) {
         logger.debug(`Adding ${yarnLockFile.name}`);
         commitFiles.push(yarnLockFile);
```

Comment:
Since you're effectively checking for `npm`>=5, should this section also be within a try/catch in case the user doesn't have `yarn` in their environment?